test/module/irohad/model/converters/pb_commands_test.cpp

```diff
@@ -74,7 +74,7 @@ TEST(CommandTest, add_peer) {
   auto factory = iroha::model::converters::PbCommandFactory();
 
   auto proto_add_peer = factory.serializeAddPeer(orig_addPeer);
-  auto serial_addPeer = factory.deserializeAddPeer(proto_add_peer);
+  const auto& serial_addPeer = factory.deserializeAddPeer(proto_add_peer);
```

Comment:
Why the result of `deserialize` method is assigned to variable named `serial...`. It is opposite - it is `deserialized`.